data/moves.js

```diff
@@ -11564,8 +11564,9 @@ let BattleMovedex = {
 		onHit(target, pokemon) {
 			let averagehp = Math.floor((target.hp + pokemon.hp) / 2) || 1;
 			target.sethp(averagehp);
+			this.add('-sethp', target, target.getHealth, '[from] move: Pain Split');
```

Comment:
```suggestion
			this.add('-sethp', target, target.getHealth, '[from] move: Pain Split', '[simult]');
```

Reply:
I wonder if it should be necessary to specify which is the attacker and which is the defender.

Probably one needs to be silent.

Reply:
Played it out locally, it seems like the animation happens once, but I get messages twice:
```
Gengar used Pain Split!
The battlers shared their pain!
The battlers shared their pain!
```

Reply:
Yeah, I would just add `[silent]` to the first `-sethp`.

Reply:
Yeah, that seems to work.